tools/c7n_azure/c7n_azure/actions.py

```diff
@@ -74,3 +94,118 @@ def process(self, resources):
                 az_resource.tags = tags
 
                 client.resources.create_or_update_by_id(resource['id'], api_version, az_resource)
+
+
+class AutoTagUser(BaseAction):
+    """Attempts to tag a resource with the first user who created/modified it.
+
+    .. code-block:: yaml
+
+      policies:
+        - name: azure-auto-tag-creator
+          resource: azure.resourcegroup
+          description: |
+            Tag all existing resource groups with the 'CreatorEmail' tag
+          actions:
+           - type: auto-tag-user
+             tag: CreatorEmail
```

Comment:
Can you also paste this into a sample policy in the documentation? Just copy one of the existing ones in the docs\azure\usecases folder.